docs/integration_resources/software/nuke.md

```diff
@@ -14,59 +14,107 @@ nshift.show()
+|:-------------|:---------------|:-------------------------------------------------|:-------------------------------------------------------------|:----------------------------------------------------------------------------------------------------|
+| *Bool*       | -              | Knob                                             | Boolean_Knob                                                 |                                                                                                     |
+| *Color*      | -              | Knob                                             | AColor_Knob                                                  |                                                                                                     |
+| *Dict*       | -              | Knob                                             | EvalString_Knob                                              | Require custom implementation                                                                       |
+| *Dir*        | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *Enumerator* | -              | Knob                                             | Enumeration_Knob                                             |                                                                                                     |
+| *FileIn*     | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *FileOut*    | -              | Knob                                             | File_Knob                                                    |                                                                                                     |
+| *Float*      | -              | Knob                                             | Double_Knob                                                  |                                                                                                     |
+| *Instance*   | image*         | Input/Output<br/>Knob<br/>Knob                   | - <br/>Channel_Knob <br/>File_Knob                           | Provides the input image like a single image. (Current Frame)                                       |
+| *Instance*   | images*        | Output<br/>Knob<br/>Knob                         | - <br/>File_Knob                                             | Provides the input image like a single image. (Current Frame)                                       |

```

Comment:
Please update the Note on this type of plug

```suggestion
| *Instance*   | images*        | Output<br/>Knob<br/>Knob                         | - <br/>File_Knob                                             | Provides the input image like a single image. (Current Frame)                                       |

```

Reply:
Please review the suggestion here https://github.com/Inbibo/shift_docs/pull/93/files#r1882338295

Reply:
commited